src/main/java/com/synopsys/integration/alert/web/user/UserActions.java

```diff
@@ -0,0 +1,88 @@
+/**
+ * blackduck-alert
+ *
+ * Copyright (c) 2019 Synopsys, Inc.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.synopsys.integration.alert.web.user;
+
+import java.util.Collection;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import javax.transaction.Transactional;
+
+import org.springframework.stereotype.Component;
+
+import com.synopsys.integration.alert.common.descriptor.accessor.AuthorizationUtility;
+import com.synopsys.integration.alert.common.exception.AlertDatabaseConstraintException;
+import com.synopsys.integration.alert.common.persistence.accessor.UserAccessor;
+import com.synopsys.integration.alert.common.persistence.model.UserModel;
+import com.synopsys.integration.alert.web.model.UserRoleModel;
+
+@Component
+@Transactional
+public class UserActions {
+    private AuthorizationUtility authorizationUtility;
+    private UserAccessor userAccessor;
+
+    public UserActions(AuthorizationUtility authorizationUtility, UserAccessor userAccessor) {
+        this.authorizationUtility = authorizationUtility;
+        this.userAccessor = userAccessor;
+    }
+
+    public Collection<UserRoleModel> getUsers() {
+        return userAccessor.getUsers().stream()
```

Comment:
Does `userAccessor.getUsers()` include the roles?

Reply:
Yes the UserModel returned by userAccessor.getUsers() contains the role names associated with the user.  We have two classes named UserRoleModel in different packages. That may cause confusion.  One is the definition of the Role and lives under the persistence package.  So it should be named RoleModel.